src/garage/tf/baselines/gaussian_conv_baseline_with_model.py

```diff
@@ -0,0 +1,67 @@
+"""Gaussian Conv Baseline."""
+import numpy as np
+
+from garage.misc.overrides import overrides
+from garage.np.baselines import Baseline
+from garage.tf.regressors import GaussianConvRegressorWithModel
+
+
+class GaussianConvBaselineWithModel(Baseline):
+    """A value function using Gaussian Conv network."""
```

Comment:
which fits to a Gaussian model whose parameters are estimated by a CNN.

Reply:
will change